src/main/java/subway/valid/InputValidator.java

```diff
@@ -0,0 +1,125 @@
+package subway.valid;
+
+import subway.domain.Line;
+import subway.domain.Station;
+
+import java.util.*;
+
+import static subway.controller.SubwayController.stationMap;
+import static subway.domain.Line.getEqualLine;
+import static subway.domain.LineRepository.lines;
+import static subway.domain.StationRepository.stations;
+import static subway.util.ErrorMessage.*;
+
+public class InputValidator {
+    public String validateMainFunction(String input) {
+        if (input.equals("1") || input.equals("2") || input.equals("3") || input.equals("4") || input.equals("Q")) {
```

Comment:
```suggestion
        if (Patterns.matches("^[1-4|Q]{1}$", input) {
```
혹시 여유가 되신다면! 정규표현식 고려해봐도 좋을 것 같아요. 괜히 헷깔릴 것 같으면 안본척하고 넘어가세요!!